doc/ins_sign_catalyst_registration.md

```diff
@@ -0,0 +1,89 @@
+# Catalyst Voting Key Registration
+
+## Description
+
+Cardano uses a sidechain for its treasury system. One needs to "register" to participate on this sidechain by submitting a registration transaction on the mainnet chain. This CIP details the registration transaction format.
+
+A Catalyst registration transaction are a regular Cardano transaction with a specific transaction metadata associated with it.
```

Comment:
I would keep it more technical here, the metadata are actually part of the tx, not just somehow "associated". And perhaps mention that the metadata is independent of tx body.